### PR TITLE
Switch from IndexOf(string) to Contains(string) for better readability and performance

### PR DESCRIPTION
**Pull request by siriak**
A small tweak to improve code a bit

### CODE REVIEW

**Comment by stephentoub** [comment]
The perf impact here on asserts is inconsequential, but this is fine from a readability standpoint. Thanks.

File: src/System.Data.Common/tests/System/Data/DataTableTest.cs [hunk 1]
```diff
@@ -3776,12 +3776,12 @@ public void WriteXmlSchema_ForignKeyConstraint()
             StringWriter sw1 = new StringWriter();
             ds1.Tables[0].WriteXmlSchema(sw1);
             string xml1 = sw1.ToString();
-            Assert.True(xml1.IndexOf(@"<xs:unique name=""Constraint1"">") != -1);
+            Assert.True(xml1.Contains(@"<xs:unique name=""Constraint1"">"));
```
Comment (danmosemsft) [inline src/System.Data.Common/tests/System/Data/DataTableTest.cs]: what about `Assert.Contains (@"<xs:unique name=""Constraint1"">", xml1));`

Comment (danmosemsft) [reply src/System.Data.Common/tests/System/Data/DataTableTest.cs]: Actually, that uses current culture. You either have to pass the StringComparison explicitly, or use `AssertExtensions.Contains` which uses Ordinal.

File: src/System.Data.Common/tests/System/Data/DataTableTest.cs [hunk 2]
```diff
@@ -3776,12 +3776,12 @@ public void WriteXmlSchema_ForignKeyConstraint()
...
 
             StringWriter sw2 = new StringWriter();
             ds1.Tables[1].WriteXmlSchema(sw2);
             string xml2 = sw2.ToString();
-            Assert.True(xml2.IndexOf(@"<xs:unique name=""Constraint1"">") == -1);
+            Assert.False(xml2.Contains(@"<xs:unique name=""Constraint1"">"));
```
Comment (danmosemsft) [inline src/System.Data.Common/tests/System/Data/DataTableTest.cs]: There is no `AssertExtensions.DoesNotContain` but you can use `Assert.DoesNotContain(@"<xs:unique name=""Constraint1"">", xml2,  StringComparison.Ordinal)`

What you have is fine, too

Comment (danmosemsft) [reply src/System.Data.Common/tests/System/Data/DataTableTest.cs]: Or feel free to add `AssertExtensions.DoesNotContain`  in AssertExtensions.cs if you like.

File: src/System.Data.Common/tests/System/Data/DataTableTest.cs (hunk 1 above)
Comment (siriak) [reply src/System.Data.Common/tests/System/Data/DataTableTest.cs]: Cool, didn't know about those methods.

File: src/System.Data.Common/tests/System/Data/DataTableTest.cs (hunk 2 above)
Comment (siriak) [reply src/System.Data.Common/tests/System/Data/DataTableTest.cs]: Updated accordingly to your first suggestion.

**Pull request merged**